edu.mit.Scratch.yaml

```diff
@@ -85,21 +81,33 @@ modules:
       - type: file
         path: scratch-desktop/node_modules/scratch-gui/package-lock.json
         dest-filename: scratch-gui-package-lock.json
+      - type: archive
+        url: https://downloads.scratch.mit.edu/microbit/scratch-microbit-1.2.0.hex.zip
+        dest: scratch-gui-src-generated
+        sha256: dfd574b709307fe76c44dbb6b0ac8942e7908f4d5c18359fae25fbda3c9f4399
+      - type: inline
+        dest: scratch-gui-src-generated
+        dest-filename: microbit-hex-url.cjs
```

Comment:
I'm sorry to report that this extension doesn't work, but that's not a regression.